commit 271c5a5d546ba6daecf101d6cf89ea162f2a1e1d
Author: Jan Kiszka <jan.kiszka@siemens.com>
Date:   Thu Jan 24 09:28:59 2019 +0100

    arm64: dts: zcu100-revC: Give wifi some time after power-on
    
    commit 35a4f89cd4731ac6ec985cd29ddc1630903006b7 upstream.
    
    Somewhere along recent changes to power control of the wl1831, power-on
    became very unreliable on the Ultra96, failing like this:
    
    wl1271_sdio: probe of mmc2:0001:1 failed with error -16
    wl1271_sdio: probe of mmc2:0001:2 failed with error -16
    
    After playing with some dt parameters and comparing to other users of
    this chip, it turned out we need some power-on delay to make things
    stable again. In contrast to those other users which define 200 ms,
    Ultra96 is already happy with 10 ms.
    
    Fixes: 5869ba0653b9 ("arm64: zynqmp: Add support for Xilinx zcu100-revC")
    Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
    Acked-by: Ulf Hansson <ulf.hansson@linaro.org>
    Signed-off-by: Michal Simek <michal.simek@xilinx.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
index eb5e8bddb610..8954c8c6f547 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
@@ -101,6 +101,7 @@
 	sdio_pwrseq: sdio_pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		reset-gpios = <&gpio 7 GPIO_ACTIVE_LOW>; /* WIFI_EN */
+		post-power-on-delay-ms = <10>;
 	};
 };
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1126_271c5a5d546ba6daecf101d6cf89ea162f2a1e1d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1126_271c5a5d546ba6daecf101d6cf89ea162f2a1e1d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
